Pipfile

```diff
@@ -4,21 +4,23 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-tox = '>=3'
```

Comment:
This is a fix independent of the move to Python 3.8. It was easiest to include here, since the `Pipfile.lock` was being updated anyway.

Reply:
Personally, I don't even have Python 3.7 available as it isn't in any of the debian repositories. For others, without a recent version of python, I agree that the docker container is adequate.